src/routes/results.js

```diff
@@ -4,12 +4,17 @@ const router = express.Router();
 const pool = require('../models/DbConnection');
 
 router.post('', async (req, res) => {
-  const { result: typeId } = req.body;
-  if (!typeId || Number(typeId) > 12 || Number(typeId) < 1) {
+  const { result: typeId, isNewResult: isNewResult } = req.body;
```

Comment:
확실한 건 아닌데 구조분해 할당이라 아마 isNewResult는 이름이 똑같으면 굳이 콜론 뒤에 안 붙여도 괜찮을 거예요!
```suggestion
  const { result: typeId, isNewResult } = req.body;
```
작동에는 문제가 없어서 고치고 싶으시다면 편하게 커밋 다시 푸시해주세요 :)

Reply:
콜론 안붙여도 되네요! 이렇게 수정할게요~